swagger/components/schemas/routerTransform.yaml

```diff
@@ -0,0 +1,58 @@
+title: Router Transform
+type: array
+items:
+  type: object
+  properties:
+    batchedRequest:
+      type: object
+      properties:
+        version:
+          type: string
+        type:
+          type: string
+        method:
+          type: string
+        endpoint:
+          type: string
+        userId:
+          type: string
+        headers:
+          type: object
+          properties: {}
+        params:
+          type: object
+          properties: {}
+        body:
+          type: object
+          properties:
+            JSON:
+              type: object
+              properties: {}
+            JSON_ARRAY:
+              type: object
+              properties: {}
+            XML:
+              type: object
+              properties: {}
+            FORM:
+              type: object
+              properties: {}
+        files:
+          type: object
+          properties: {}
+        metadata:
+          $ref: './metadata.yaml'
```

Comment:
This is not going the be used here (not needed only in delivery response it is used)

Reply:
Remvoed